docs/DesignDocs/AttestationQPLEnforcementSupport.md

```diff
@@ -0,0 +1,95 @@
+Attestation SGX QPL Enforcement Support
+====
+
+This proposal is to make a change to `oe_verify_evidence`, so that it can support policy-based SGX QPL (Quote Provider Library) specific enhancement, such as allowing the API user to pass data to the SGX QPL for more advanced quote verification.
+
+Motivation
+----
+
+Intel SGX rolls out PCK certificate and collateral information about every six months.
+There is a scenario where a quote signed by the last version of PCK certificate cannot be verified with the latest version of collateral information.
+This causes unnecessary attestation failures when new versions of PCK certificate and collateral information are rolled out.
+To deal with this issue, the SGX QPL need to return any version of collateral information based on additional query parameters (referred to as "baselines").
```

Comment:
```suggestion
To deal with this issue, the SGX QPL needs to return any version of collateral information based on additional query parameters (referred to as "baselines").
```
grammar nit

Reply:
Thanks for this, I will read through it again, really appreciate this!